cmd/konnectd/bootstrap.go

```diff
@@ -59,12 +58,34 @@ const (
 	apiTypeSignin  = "signin"
 )
 
-// bootstrap is a data structure to hold configuration required to start
-// konnectd.
-type bootstrap struct {
-	cmd  *cobra.Command
-	args []string
+//Stringly typed application config, represents the user accessible config params
```

Comment:
Guess we need some travis integration running on githhub pr's. I know that Konnect's linting state is not in a very good state, but please try to avoid adding new issues if possible :)